src/components/Activities/Activity1/Game2/components/GameSlide.tsx

```diff
@@ -30,15 +32,26 @@ function GameSlide(props: GameSlideProps): JSX.Element {
     return props.textDefault ? props.textDefault : '';
   };
 
+  const handleAlienState = (state: ALIEN_STATE) => {
+    clearTimeout(alienTimeout.current);
+    setAlienState(state);
+    alienTimeout.current = setTimeout(() => {
+      alienTimeout.current = undefined;
```

Comment:
(2) here also if we wanna do this/wanna do now 
or we can do in another method/rename to avoid that its surpirsing for handleAlienState to handle advancing the game also
```suggestion
      alienTimeout.current = undefined;
       if(state === ALIEN_STATE.HAPPY)
        props.advanceGame && props.advanceGame();
```

Reply:
Yeah i think having the handleAlienState should just handle how the alien functions

I just made it it's own function Bc I was gonna reuse the code of the timeouts for both anger and happy